### ISSUE TITLE
TypeError: _0x2f33cc[(_0x1efd68(...) + _0x1efd68(...))] is not a function

### ISSUE THREAD

**Issue by anoduck** [microsoft/pylance-release#5074]
## Environment data

-   Language Server version: v2023.10.50
-   OS and version: Kali Linux - Rolling Release
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.11
-   Poetry version: 1.6.1

## Code Snippet

```python
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

import os
import sys
sys.path.append(os.path.expanduser('~/.local/lib/python3.11/site-packages'))

print('I am a sample piece of code that stops pylance from working.')

```

## Actual behavior

Without any noticed significant changes to the system, pylance started reporting that the server had crashed five time in the last three minutes and would not be restarting. 

## Logs

```
2023-11-02 14:21:06.226 [info] [Info  - 2:21:06 PM] (75143) Pylance language server 2023.10.50 (pyright cfe56bfb) starting
2023-11-02 14:21:06.230 [info] [Info  - 2:21:06 PM] (75143) Server root directory: /home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist
2023-11-02 14:21:06.263 [info] [Info  - 2:21:06 PM] (75143) Starting service instance "<default>"
2023-11-02 14:21:06.713 [info] (75143) No pyproject.toml file found.
2023-11-02 14:21:06.723 [info] [Info  - 2:21:06 PM] (75143) Setting pythonPath for service "<default>": "/usr/bin/python3"
2023-11-02 14:21:06.733 [info] [Info  - 2:21:06 PM] (75143) Setting environmentName for service "<default>": "3.11.6 (global)"
2023-11-02 14:21:07.128 [info] [Info  - 2:21:07 PM] (75143) Assuming Python version 3.11
2023-11-02 14:21:07.128 [info] (75143) Assuming Python platform Linux
2023-11-02 14:21:08.009 [info] [Info  - 2:21:08 PM] (75143) Search paths for <default>
2023-11-02 14:21:08.010 [info] [Info  - 2:21:08 PM] (75143)   /home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/typeshed-fallback/stdlib
2023-11-02 14:21:08.011 [info] [Info  - 2:21:08 PM] (75143)   typings
2023-11-02 14:21:08.012 [info] [Info  - 2:21:08 PM] (75143)   /home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/typeshed-fallback/stubs/...
2023-11-02 14:21:08.012 [info] [Info  - 2:21:08 PM] (75143)   /home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/bundled/stubs
2023-11-02 14:21:08.014 [info] [Info  - 2:21:08 PM] (75143)   /usr/lib/python3.11
2023-11-02 14:21:08.015 [info] [Info  - 2:21:08 PM] (75143)   /usr/lib/python3.11/lib-dynload
2023-11-02 14:21:08.015 [info] [Info  - 2:21:08 PM] (75143)   /home/user/.local/lib/python3.11/site-packages
2023-11-02 14:21:08.016 [info] [Info  - 2:21:08 PM] (75143)   /usr/local/lib/python3.11/dist-packages
2023-11-02 14:21:08.016 [info] [Info  - 2:21:08 PM] (75143)   /usr/local/lib/python3.11/dist-packages/loguru-0.7.0-py3.11.egg
2023-11-02 14:21:08.018 [info] [Info  - 2:21:08 PM] (75143)   /usr/local/lib/python3.11/dist-packages/isc_dhcp_leases-0.10.0-py3.11.egg
2023-11-02 14:21:08.021 [info] [Info  - 2:21:08 PM] (75143)   /usr/local/lib/python3.11/dist-packages/cryptography-41.0.1-py3.11-linux-x86_64.egg
2023-11-02 14:21:08.026 [info] [Info  - 2:21:08 PM] (75143)   /usr/local/lib/python3.11/dist-packages/pyOpenSSL-23.3.0.dev0-py3.11.egg
2023-11-02 14:21:08.046 [info] [Info  - 2:21:08 PM] (75143)   /usr/lib/python3/dist-packages
2023-11-02 14:21:08.052 [info] [Info  - 2:21:08 PM] (75143)   /usr/lib/python3.11/dist-packages
2023-11-02 14:21:08.059 [info] [Info  - 2:21:08 PM] (75143) Adding fs watcher for library directories:
 /usr/lib/python3.11
/usr/lib/python3.11/lib-dynload
/home/user/.local/lib/python3.11/site-packages
/usr/local/lib/python3.11/dist-packages
/usr/local/lib/python3.11/dist-packages/loguru-0.7.0-py3.11.egg
/usr/local/lib/python3.11/dist-packages/isc_dhcp_leases-0.10.0-py3.11.egg
/usr/local/lib/python3.11/dist-packages/cryptography-41.0.1-py3.11-linux-x86_64.egg
/usr/local/lib/python3.11/dist-packages/pyOpenSSL-23.3.0.dev0-py3.11.egg
/usr/lib/python3/dist-packages
/usr/lib/python3.11/dist-packages
2023-11-02 14:21:08.059 [info] (75143) Searching for source files
2023-11-02 14:21:08.067 [info] [Info  - 2:21:08 PM] (75143) No source files found.
2023-11-02 14:21:08.076 [info] (75143) pytest configurations: {"message":"script","classes":["Test"],"files":["test_*.py","*_test.py"],"functions":["test"]}
2023-11-02 14:21:08.080 [info] (75143) pytest configurations: {"message":"script","classes":["Test"],"files":["test_*.py","*_test.py"],"functions":["test"]}
2023-11-02 14:21:08.418 [info] TypeError: _0x2f33cc[(_0x1efd68(...) + _0x1efd68(...))] is not a function
    at _0xa6e677.dirExistsCached (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:902382)
    at _0xa6e677._resolveAbsoluteImport (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:303694)
    at _0xa6e677.resolveAbsoluteImport (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:296859)
    at _0xa6e677._resolveBestAbsoluteImport (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:305905)
    at _0xa6e677._resolveImportStrict (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:299332)
    at _0xa6e677.resolveImportInternal (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:294521)
    at _0xa6e677.resolveImport (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:290200)
    at _0xa6e677.resolveImport (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:898287)
    at _0x170d92.<computed> (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:272928)
    at _0x5e13d0._enableJinjaTelemetry (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:1078110)
    at _0x5e13d0.updateSettingsForWorkspace (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:1019982)

2023-11-02 14:21:08.429 [info] (75143) Server settings returned for workspace: : {
  "autoSearchPaths": true,
  "disableLanguageServices": false,
  "openFilesOnly": true,
  "useLibraryCodeForTypes": true,
  "watchForSourceChanges": true,
  "watchForLibraryChanges": true,
  "watchForConfigChanges": true,
  "typeCheckingMode": "off",
  "diagnosticSeverityOverrides": {
    "reportShadowedImports": "warning"
  },
  "logLevel": "log",
  "autoImportCompletions": false,
  "includeUserSymbolsInAutoImport": true,
  "indexing": false,
  "completeFunctionParens": true,
  "enableExtractCodeAction": true,
  "indexOptions": {
    "packageDepths": [
      [
        "sklearn",
        2,
        false
      ],
      [
        "matplotlib",
        2,
        false
      ],
      [
        "scipy",
        2,
        false
      ],
      [
        "django",
        2,
        false
      ],
      [
        "flask",
        2,
        false
      ],
      [
        "fastapi",
        2,
        false
      ]
    ],
    "localStoragePath": "",
    "userFileIndexingLimit": 2000
  },
  "variableInlayTypeHints": false,
  "callArgumentNameInlayHints": "off",
  "functionReturnInlayTypeHints": false,
  "pytestParametersInlayTypeHints": false,
  "importFormat": "absolute",
  "fileSpecs": [],
  "excludeFileSpecs": [],
  "ignoreFileSpecs": [],
  "formatOnType": true,
  "taskListTokens": [],
  "addImportExactMatchOnly": false,
  "enablePytestSupport": true,
  "enablePytestExtra": false,
  "gotoDefinitionInStringLiteral": true,
  "remapDiagnostics": false,
  "intelliCodeEnabled": true,
  "pythonPath": "/usr/bin/python3",
  "pythonEnvironmentName": "3.11.6 (global)",
  "fixAll": [],
  "extraCommitChars": true,
  "notebookRunStartupCommands": "",
  "editorIndentConfig": {
    "defaultTabSequence": "    ",
    "tabSize": 4
  }
}
2023-11-02 14:21:08.451 [info] (75143) No pyproject.toml file found.
2023-11-02 14:21:08.451 [info] [Info  - 2:21:08 PM] (75143) Setting pythonPath for service "<default>": "/usr/bin/python3"
2023-11-02 14:21:08.452 [info] [Info  - 2:21:08 PM] (75143) Setting environmentName for service "<default>": "3.11.6 (global)"
2023-11-02 14:21:08.775 [info] [Info  - 2:21:08 PM] (75143) Assuming Python version 3.11
2023-11-02 14:21:08.775 [info] (75143) Assuming Python platform Linux
2023-11-02 14:21:09.106 [info] [Info  - 2:21:09 PM] (75143) Search paths for <default>
2023-11-02 14:21:09.110 [info] [Info  - 2:21:09 PM] (75143)   /home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/typeshed-fallback/stdlib
2023-11-02 14:21:09.115 [info] [Info  - 2:21:09 PM] (75143)   typings
2023-11-02 14:21:09.125 [info] [Info  - 2:21:09 PM] (75143)   /home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/typeshed-fallback/stubs/...
2023-11-02 14:21:09.133 [info] [Info  - 2:21:09 PM] (75143)   /home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/bundled/stubs
2023-11-02 14:21:09.134 [info] [Info  - 2:21:09 PM] (75143)   /usr/lib/python3.11
2023-11-02 14:21:09.134 [info] [Info  - 2:21:09 PM] (75143)   /usr/lib/python3.11/lib-dynload
2023-11-02 14:21:09.134 [info] [Info  - 2:21:09 PM] (75143)   /home/user/.local/lib/python3.11/site-packages
2023-11-02 14:21:09.135 [info] [Info  - 2:21:09 PM] (75143)   /usr/local/lib/python3.11/dist-packages
2023-11-02 14:21:09.135 [info] [Info  - 2:21:09 PM] (75143)   /usr/local/lib/python3.11/dist-packages/loguru-0.7.0-py3.11.egg
2023-11-02 14:21:09.141 [info] [Info  - 2:21:09 PM] (75143)   /usr/local/lib/python3.11/dist-packages/isc_dhcp_leases-0.10.0-py3.11.egg
2023-11-02 14:21:09.142 [info] [Info  - 2:21:09 PM] (75143)   /usr/local/lib/python3.11/dist-packages/cryptography-41.0.1-py3.11-linux-x86_64.egg
2023-11-02 14:21:09.142 [info] [Info  - 2:21:09 PM] (75143)   /usr/local/lib/python3.11/dist-packages/pyOpenSSL-23.3.0.dev0-py3.11.egg
2023-11-02 14:21:09.142 [info] [Info  - 2:21:09 PM] (75143)   /usr/lib/python3/dist-packages
2023-11-02 14:21:09.142 [info] [Info  - 2:21:09 PM] (75143)   /usr/lib/python3.11/dist-packages
2023-11-02 14:21:09.142 [info] [Info  - 2:21:09 PM] (75143) Adding fs watcher for library directories:
 /usr/lib/python3.11
/usr/lib/python3.11/lib-dynload
/home/user/.local/lib/python3.11/site-packages
/usr/local/lib/python3.11/dist-packages
/usr/local/lib/python3.11/dist-packages/loguru-0.7.0-py3.11.egg
/usr/local/lib/python3.11/dist-packages/isc_dhcp_leases-0.10.0-py3.11.egg
/usr/local/lib/python3.11/dist-packages/cryptography-41.0.1-py3.11-linux-x86_64.egg
/usr/local/lib/python3.11/dist-packages/pyOpenSSL-23.3.0.dev0-py3.11.egg
/usr/lib/python3/dist-packages
/usr/lib/python3.11/dist-packages
2023-11-02 14:21:09.142 [info] (75143) Searching for source files
2023-11-02 14:21:09.142 [info] [Info  - 2:21:09 PM] (75143) No source files found.
2023-11-02 14:21:09.281 [info] [Error - 2:21:09 PM] (75143) An internal error occurred while parsing file "untitled:Untitled-1": TypeError: _0x2f33cc[(_0x1efd68(...) + _0x1efd68(...))] is not a function
    at _0xa6e677.dirExistsCached (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:902382)
    at _0xa6e677._resolveAbsoluteImport (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:303694)
    at _0xa6e677.resolveAbsoluteImport (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:296859)
    at _0xa6e677._resolveBestAbsoluteImport (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:305905)
    at _0xa6e677._resolveImportStrict (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:299332)
    at _0xa6e677.resolveImportInternal (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:294521)
    at _0xa6e677.resolveImport (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:290200)
    at _0xa6e677.resolveImport (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:898287)
    at i (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:510945)
    at _0x4c3a07._resolveImports (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:511197)
    at /home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:499381
    at a.timeOperation (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:1130032)
    at /home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:499348
    at t.LogTracker.log (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:1092230)
    at _0x4c3a07.parse (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:498098)
    at N._parseFile (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:434006)
    at N._bindFile (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:434964)
    at N.getBoundSourceFileInfo (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:420286)
    at N.getBoundSourceFile (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:420039)
    at k.getSourceFile (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/pyright.bundle.js:1:467417)
    at _0x52d99f.<computed>._runCore (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:255884)
    at _0x52d99f.<computed>.runWithPosition (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:255632)
    at /home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:1037836
    at _0x5f4a97.<computed> (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:367551)
    at _0x5e13d0.onCompletion (/home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1:1037605)
2023-11-02 14:21:09.303 [info] (75143) [FG] parsing: untitled:Untitled-1 (79ms)
2023-11-02 14:21:09.312 [info] (75143) [FG] binding: untitled:Untitled-1 (6ms)
2023-11-02 14:21:09.394 [info] 

2023-11-02 14:21:09.448 [info] /home/user/.vscode/extensions/ms-python.vscode-pylance-2023.10.50/dist/server.bundle.js:1
function a1_0x2111(){const _0x3485db=['CY4IFsWIuMvMyq','zgL0Aw9UywXfCG','ywjSzsi6iKrLXj8','AgfUz2vK','ugf0Aci6iLTRqG','x2DLDerVy3vTzq','q292zxjHz2u','Bwv0Ag9Ksw52BW','y2HKAxi','C2nPChK','oIlRQQJRKzaGuhLSyw4','igLUBgf5sgLUDa','sw5fEhbLCMLTzq','B1jLBgf0AxzLua','DxjL','BguIoIlqN9c10ydqTDc8','C09UBhK','lLrVz2DSzuLTCa','Aw9Ut3b0Aw9UCW','EhrmAw5L','x2fWCgX5uhLSyq','z2v0uMvHBezPBa','zNqGC3rVCMuIFq','BgXLiezPEhr1CG','x25VzgvdB250yq','ywnOic4UlIiSiG','Dmo4imoREhrYW6zqW6y','44gz44kliN0SiLvUCMu','44cciN0SiK1LC3nH','zwrPDa','CgfYC2uG','igvUy29UDhjHCG','Cg9ZAxrPB25bCG','x2nSyxnZrgvJBa','C2vYDMvYt3b0Aq','vgv4DevKAxq','BwvUDe5HBwuIoG','zgvSAw1PDgvY','xciGBMLJAhqGzW','ChrLzenVBw1HBG','DgvZigXLCYbPBq','C2LTza','CMf0B3i','ifWIE29Sze1Vza','CNrLigLUihjLBa','sw5KzxHPBMCGra','u3LTyM9SihzLCG','z2vYzsbHBM5VDa','EYjTB3zLrMLSzq','x2fWCgvUzeDLDa','lMnHy2HL','CIbCiNTUyw1LFq','yJrImZe2zwvLza','zxH0zw5ZAw9Uva','z2v0rMLYC3rbBG','z2v0uhjVz3jHBq','AgvZAxntCgfUuW','CSoRDmoRCUg6PmsF5ycQXldqGG','ihrLC3rLignVBG','vg9Rzw4','t3zLCMXVywrvCW','y3jHC2H0zxn0','x3nLCNzPy2vjza','zg93C1n0B3jLrq','C2LKzq','ihrVihjLBgf0Aq','Dhj5','zwXZzvn1AxrL','z2v0uhL0Ag9UrG','sw5KzxHLCIbJyq','Bg9JyxrPB25Z','CNrjBMzVrM9Yva','lxmZ','BNvTyMvY','DxjLrMLUzgvY','DMvYihpdRw1IB2W','t2zMC2v0','yxrLvhLWzvn0Dq','x3jVB3rvCMK','CMvHzezPBgvtEq','z2v0u2vSzwn0zq','zxn0','C29SDxrL','y3jLyxrLq2fUyW','BIbKAwqGBM90ia','zuLUrNjVBuLTCa','BsbPW6DLCMKGywS','zgvZvg9tDhjPBG','z2v0rML4DhvYzq','C1rPBwvjBK1t','CIbSB29W','zfrVrxzLBNq','x2DLDe1HDgnOAq','CgvYzM9YBwfUyW','C3r1yNm','Bw92zuzPBgvmyq','yw1LFEkaNEEAHoAjGoACIEwVVowfPq','z2v0ugvYC2LZDa','yxjLihr1DhrPia','x19IExrLC19F','rxjYB3i','rgvZy3jPChrPBW','qNL0zxm','BguGCgf0AcbKBW','y2XVBMvpCLnLDa','x3n0B3bxyxrJAa','zuz1BMn0Aw9U','DhjHy2TqzxjM','y2vYDgLMAq','DhLWzufYz3vTzq','vKvsu0LptG','CNrHEMLVBMKGAq','igtdTM7dVmwFDmo8CIi','B2X2zwrjBxbVCG','zxHWzwn0zwrpCa','BgLUzxm','q2fJAgu','AM9PBG','AeXVz0XLDMvS','iKnOy2v0zsbHAW','zvjLBMfTzq','z2v0tM90zwjVBW','v2fYBG','zgf0ywrVzW','ieKGBwv0B2rPia','CYbMB3jTyxrVCW','x2rPywDUB3n0Aq','Dg90ywXnCW','x2DLDeXPBMvfBG','ExbLv2L0Aef1Da','zenVBNn0CNvJDa','zSo4CIddPMXSigBdRW','CM9VDfbHDgG','CIb0B2rHCYbSyq','Bw9KDwXLrMLSzq','0lRqUcdqVDcWifWIE28','Cg9LDhj5','x2r1BxbgAwXLra','Bwv94OczioY2LoQWGcj9la','ywDLugf0Aa','Bg9HzgvK','A3r1ywXPC2LLCG','Bg9Zzxn0tM9Kzq','CMTZCgfJzuzPBa','BM9ZDgLJu2LUAW','sNvWExrLCLjLzG','C8oTDMvSigXVy2e','Aw5KB3DZu3rVCG','A2vUswzuExbL','4kwn4ksK4ks/4kwd44g+4lIz4lIX4lMj4ksI4kwc','Ag9UvMvYC2LVBG','ioEoR+wIGYiSiMDSB2i','AcbSDwiGCgfYyq','lcjTB3zLu3LTyG','x2nSzwfYugvYCW','yCoNW7vLCYbUW6nVia','rNvUy3rPB25bBG','AxnkAw5QyuzPBa','BwLZC2LUz0LTCa','t25mAw5L','zxHLy0vUDG','ywn0Aw9U','BMfTzuLUzM8','z2v0vw5LC2nHCa','DwqTDMLKzw9PBG','zNv0DxjLsw1WBW','ywrKq3vZDg9Ttq','uMvTB3zLiefSBa','zwDPB25dB21Tzq','CNrZqwrKzwq','ihvUigf1DhjLia','iK1VDMvYihpdRw0','AgfZuhL0zxn0','ywnOzwq','yxvZD8oKAgXLBIi','yxP1CMuTC2vYDG','y29UDMvYDefSBa','rMfPBgvKihrVia','vhLWzujHC2u','CNLqyxrO','zgLY','AwDUB3jL','D2XPBMvpCLnLBq','zwXZzq','CNjLza','DgLVBNm6ia','ChL0Ag9UlNnLDa','x2nHBMnLBgXHDa','AcbWCMvMAxHdzq','yw5ZzM9YBwvY','vMfYqxjNtgLZDa','iLvUCMvZB2X2zq','uMvHBfrLBxbgAq','vhvWBgvdBgfZCW','zvn5BwjVBhnuBW','zxjYBM8','zxH0CMfdB21TAq','Dg9Yzsj9Fq','B2nRlcbVCIbTBW','BM5LBa','y3rLCG','rxHWzxjPBwvUDa','A3nWywnLu3vWCa','BNrdB25MAwC','sw1WB3j0qwXPyq','CgvZ','CgfYyw1LDhj5ia','AgfZq29TCgXLDa','CMvZB2X2Aw5Nia','sw1WB3j0CYi6iG','yxbWzxi','Aw9UiIWIy29UDG','zgKGAw1WB3j0yq','BSoTiIWID2LUzg8','BguGAw1WB3j0yq','B2XKtw9KDwXLtG','DMfYAwfIBgvuEq','Dxb5DgvYigrPzG','x2XHC3reAwfNBG','CM9Trw52','B2Xoyw1LC1rV','zxnZywDLCYi6EW','zw50zwq','DgvZrM9YqwjICG','zu5HBwu','zw5KCMf3','lxrHyMXLCW','z2v0v29YA3nWyq','B2WIoNSIy29TBq','CM5LzcbMB3iGDW','sw5Kzw50yxrPBW','zufZEw5J','CgfYyw1LDhjPEG','C1jLz2v4zxm','Aci6iMv4DhjHua','zerLy2W','y3vTzw50u3LUyW','y29SBgvJDa','CMvJDxjZAxzL','W6PUy2LHCYbKzsa','AxrLBxnuB01VDG','zenVBw1HBMq','C2vKsw1WB3j0lG','x2DLDeLTCg9YDa','ChjLDMLVDxnszq','vw5IB3vUza','uhjLzMvYu291CG','uhLoyunS','t3jtDgfYrxHWCG','lwf1DgHVCML6yq','4ksK4ks/4kwd44g+4BQKXj/LGkReSncc0Ra','W7f2W6TYDcb0W7GGW6y','sxnjBNrLBgXPqW','yxjPC29U','AxjVBM1LBNroyq','zCw8XivJzsdfM3jVza','qwjZB2X1Dgvqyq','DMLZAxrbC3nPzW','z2v0t3b0Aw9UCW','q29UDgfPBNndBW','zxj0igfSBcbPBq','iNTVBgrnB2r1Ba','ChrPB24IoIjBCq','Dw5WywnRvhvWBa','DezYB21pCMLNAq','x3bHCNnLtMfTzq','ChLYAwDODc5KDq','r2vUzxjPyW','Aw1WB3j0CYb0BW','AwfNBM9ZDgLJiG','vhLWzvbHCMfTzq','DefSBfrVuMvSyq','6kof572U44cc5Rwl6k+v5PA55Rov5BQu5y+Q5O6L','Dgf0Aw9UCYbPBG','ywrKsw1WB3j0sq','rMLSzujHC2vKva','D29YA3nWywnLua','DgLUz3mGCMv0Dq','0jJqT9cY0lVqTDgh0yWG0lZqTq','Aw4GDw4GBNvVDG','y2HHCNnLDc1UBW','tgvZC1rOyw4','BhnwzxjZAw9U','Bg9Nsw5MBW','xciG44gR5PU05PAW44gx44g+44gz44gl','qMfZzurPC3bHDa','swzoB2rL','zgLZywjSzunSAq','q29UDgvUDhm','y2vjBMzV','x3rYEuDLDfjLCW','icHWExjPz2H0ia','Dg9Rzw5syw5Nzq','u3rHDhvZqMfYtq','z2v0uhLuExbLza','y2HHCKnVzgvbDa','qxn5BMndB21Tyq','zgvICMf0ig5LCa','DwvoB3rbBgXVDW','zsi6iK1Py3jVCW','DhvYzq','z2v0uhL0Ag9UuW','AwDUB3jLrxjYBW','x3rYEuDLDerLyW','x2LZsw5mB29W','zgLKq2XVC2u','zNvSBa','ywrKrxH0CMfqyq','zw5XDwv1zvjLCq','ze1VzhvSzu5HBq','DgHLBG','x2HHBMrSzunSyq','zxj0qwXSvg9byG','x2fSBg93zwq','vg9Rzw5PEMvY','DefSBfrVqwjZBW','z2v0qMfJA2DYBW',

[... LOTS of output ...]

```

**Comment by debonte** [comment]
@anoduck, this is a duplicate of #5009 which is fixed in our latest prerelease build (2023.10.53).

**Comment by anoduck** [comment]
VSCode is like a drug I never want to quit. So glad this bug is covered.
